praw/objects.py

```diff
@@ -1807,8 +1809,9 @@ def __init__(self, reddit_session, subreddit=None, page=None,
             page = json_dict['page']
         info_url = reddit_session.config['wiki_page'].format(
             subreddit=six.text_type(subreddit), page=page)
+        self._params = kwargs
         super(WikiPage, self).__init__(reddit_session, json_dict, fetch,
-                                       info_url, **kwargs)
+                                       info_url)
```

Comment:
`WikiPage` is a subclass of `RedditContentObject`, and `RedditContentObject` doesn't have `**kwargs` in its constructor. There probably should be, but it didn't feel right to mess with ~10 other classes that inherit from it. None of the other classes use `**kwargs`, so why should WikiPage be the exception?

That being said, very few of those classes even have their own constructor anyway, so it's probably not as big an impact as I had in mind originally.